ui/StatusQ/src/StatusQ/Core/LocaleUtils.qml

```diff
@@ -301,10 +301,13 @@ QtObject {
 
         // otherwise
         var fullFormatString = d.fixupTimeFormatString(loc.dateTimeFormat(Locale.ShortFormat))
-        if (now.getFullYear() === value.getFullYear())
-            fullFormatString = fullFormatString.replace(/y/g, "") // strip year part, if current year -> "31 December 09:41"
-        else
+        if (now.getFullYear() === value.getFullYear()) {
+            // strip year part, if current year -> "31 December 09:41"
+            // It remove preceding dot or space
+            fullFormatString = fullFormatString.replace(/([.\s])?\b(y+)\b/g, "")
```

Comment:
The problem I encountered is that my locale was `DD.MM.yyyy`, that means when removing simply `yyyy` it left dot at the end. Generally it will be problem for any not standard format, like Swedish where year start at the beginnig.

Thanks for pointing out that there might be more cases.
I will report issue on that so I won't block this PR.

Reply:
@caybro  added #10615

Reply:
Thanks!